Kickstarter-iOS/Views/Controllers/PledgeTableViewController.swift

```diff
@@ -52,6 +53,13 @@ class PledgeTableViewController: UITableViewController {
         self?.dataSource.load(amount: amount, currency: currency)
         self?.tableView.reloadData()
     }
+
+    self.viewModel.outputs.estimatedDeliveryDate
```

Comment:
In order to prevent double reloading would it be possible to pass the `deliveryDate` as part of the other signal together with `amount` and `currency`?